src/Discord.Net.Core/Discord.Net.Core.csproj

```diff
@@ -4,13 +4,13 @@
     <AssemblyName>Discord.Net.Core</AssemblyName>
     <RootNamespace>Discord</RootNamespace>
     <Description>The core components for the Discord.Net library.</Description>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net46;netstandard1.3;netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.3;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net461;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.0;netstandard2.1</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
     <PackageReference Include="System.Collections.Immutable" Version="1.3.1" />
-    <PackageReference Include="System.Interactive.Async" Version="3.2.0" />
+    <PackageReference Include="System.Interactive.Async" Version="4.0.0" />
```

Comment:
carrying on from the above comment
```
<ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">
    <PackageReference Include="System.Interactive.Async" Version="4.0.0" />
  </ItemGroup>
  <ItemGroup Condition=" '$(TargetFramework)' != 'netcoreapp3.0' ">
    <PackageReference Include="System.Interactive.Async" Version="3.2.0" />
  </ItemGroup>
```

Reply:
If I went down that path, it couldn't be a NC3 switch as lx 4 supports NS2 & NETFX461 (a newer version of lx might be resolved through other nuget packages the user installs and screw over everything). The old version would exist purely to support NS1.3. There'd also need to be a number of ifdefs added to support the old syntax. I can do a setup like that if required but I'm not sure it's worth the effort just to support .NET Core 1.x which has been EoL for quite a long time now.